documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -276,11 +278,11 @@ Compute Modules have a dedicated `nRPIBOOT` jumper to select `RPIBOOT` mode. Fla
 * `7`
 * `8`
 
-This property does not depend on `secure-boot`. However, you should verify that this GPIO configuration does not conflict with any HATs which might pull the GPIO low during boot.
+The GPIO may be used as a general-purpose I/O pin after the OS has started. However, you should verify that this GPIO configuration does not conflict with any HATs which might pull the GPIO low during boot.
 
-For safety, this property can _only_ be programmed via `RPIBOOT`. As a result, you must first clear the bootloader EEPROM using `erase_eeprom`. This causes the ROM to failover to `RPIBOOT` mode, which then allows this option to be set.
+Although `secure-boot` requires this property to be set on Raspberry Pi 4B and Raspberry Pi 400, it does not depend on `secure-boot`. For example, `RPIBOOT` can be useful for automated tested.
 
-On BCM2712, you can alternatively force `RPIBOOT` mode by holding down the power button while simultaneously connecting a USB-C power supply.
```

Comment:
Should we keep this line in, whilst removing the word "alternatively" ?

Reply:
I'd rather get rid of the BCM2712 section entirely here because this is referring to a property for setting an OTP bit and that's not relevant there. nRPIBOOT is covered in in provisioning docs - where supported.